docs/examples/index.rst

```diff
@@ -0,0 +1,14 @@
+Examples of using MQML
+======================
+
+These are all examples based on notebooks that can be found in
+https://github.com/QCoDeS/MQML-scripts/tree/master/docs/examples and below. To experiment with the
+examples you can download them directly from the git repository.
+
+Parameter
+---------------
```

Comment:
The underline should match the length of the text

Reply:
Thanks for the review, I will amend this.